altair/utils/save.py

```diff
@@ -114,6 +114,8 @@ def save(
     **kwargs :
         additional kwargs passed to spec_to_mimebundle.
     """
+    from altair.utils._vegafusion_data import using_vegafusion
```

Comment:
Could you add a comment above this line why the import is in the function so that this does not get modified unintentionally later on? Due to circular imports? If the import could also be at the top of the file, I'd prefer that.

Reply:
Done in 6e4c8c4fc3d93e8cbd9e296ad79c11b957729eeb